integration/tests/epochs/epoch_test.go

```diff
@@ -139,7 +139,8 @@ func (s *Suite) TestEpochJoinAndLeave() {
 	role := flow.RoleAccess
 
 	// grab the first container of this node role type, this is the container we will replace
-	containerToReplace := s.net.ContainersByRole(role)[0]
+	containerToReplace := s.getContainerToReplace(role)
+	require.True(s.T(), containerToReplace != nil)
```

Comment:
```suggestion
	require.NotNil(s.T(), containerToReplace)
```
Same thing, but gives more informative error messages

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/55810e1bfedd693005ff41ef5322af68b0a6d5dd